cluster/vagrant/provision-master.sh

```diff
@@ -257,10 +257,10 @@ popd
 # Enable Fedora Cockpit on host to support Kubernetes administration
 # Access it by going to <master-ip>:9090 and login as vagrant/vagrant
 if ! which /usr/libexec/cockpit-ws &>/dev/null; then
-  
+
   pushd /etc/yum.repos.d
     wget https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/repo/fedora-21/sgallagh-cockpit-preview-fedora-21.repo
```

Comment:
s/21/22 in above line, but this will need to be branched on fedora version until all providers are updated:

```
wget https://copr.fedoraproject.org/coprs/sgallagh/cockpit-preview/repo/fedora-22/sgallagh-cockpit-preview-fedora-22.repo
```